app/views/softdrinksindustrylevy/register/litreagePage.scala.html

```diff
@@ -21,38 +21,40 @@
 
 @(litreageForm: Form[_], pageName: String, backLink: Call, nextLink: Option[Call] = None)(implicit request: Request[_], messages: Messages, config: AppConfig)
 
-    @main_template(Messages("sdil.common.title"), form = Some(litreageForm)) {
-
-        <a class="link-back" href="@backLink.url">@Messages("sdil.common.back")</a>
-
-        @error_summary(litreageForm)
-
-        <h1 class="heading-large">@Messages(s"sdil.$pageName.heading")</h1>
-
-        @form(nextLink.getOrElse(routes.LitreageController.submit(pageName)), 'novalidate -> "") {
-            <div class="form-group form-litres">
-                @inputText(
-                    litreageForm("lowerRateLitres"),
-                    Messages("sdil.litreage.lowRate"),
-                    '_inputType -> "text",
-                    '_inputClass -> "input--no-spinner volume"
-                )
-            </div>
-
-            <div class="form-group form-litres">
-                @inputText(
-                    litreageForm("higherRateLitres"),
-                    Messages("sdil.litreage.highRate"),
-                    '_inputType -> "text",
-                    '_inputClass -> "input--no-spinner volume"
-                )
-            </div>
-
-            @details("sdil.litreage.rates.help") {
-                <p>@Messages("sdil.litreage.rates.content.p1")</p>
-                <p>@Messages("sdil.litreage.rates.content.p2")</p>
-            }
-
-            <input class="button" value="@Messages("sdil.common.save-continue")" type="submit">
+@main_template(Messages("sdil.common.title"), form = Some(litreageForm)) {
+
+    <a class="link-back" href="@backLink.url">@Messages("sdil.common.back")</a>
+
+    @error_summary(litreageForm)
+
+    <h1 class="heading-large">@Messages(s"sdil.$pageName.heading")</h1>
+
+    @form(nextLink.getOrElse(routes.LitreageController.submit(pageName)), 'novalidate -> "") {
+        <div class="form-group form-litres">
+        @inputText(
+            litreageForm("lowerRateLitres"),
+            Messages("sdil.litreage.lowRate"),
+            '_inputType -> "text",
+            '_inputClass -> "input--no-spinner volume"
+        )
+        </div>
+
+        <div class="form-group form-litres">
+        @inputText(
+            litreageForm("higherRateLitres"),
+            Messages("sdil.litreage.highRate"),
+            '_inputType -> "text",
+            '_inputClass -> "input--no-spinner volume"
+        )
+        </div>
+
+        @details("sdil.litreage.rates.help") {
+            <p>@Messages("sdil.litreage.rates.content.p1")
```

Comment:
shouldn't this be two paragraphs, rather than one paragraph with a `<br>`?

Reply:
```<p>
	    The low band is from 5 grams to 7.9 grams of sugar per 100 millilitres.<br>
	    The high band is 8 grams of sugar or more per 100 millilitres.
	  </p>```
No that's what it used to be but is changed in latest prototype as shown in snippet from above.
Hold off on this PR anyway until we know what's happening

Reply:
I've seen that in the prototype, also it looks a lot better.